### PR TITLE
Call keyboard event directly without event queue

### PR DESCRIPTION
**Pull request by namse**
Same with https://github.com/NamseEnt/namseent/pull/519, keyboard version.

### CODE REVIEW

File: namui/src/namui/render/special/attach_event.rs [hunk 1]
```diff
@@ -75,7 +75,7 @@ impl WheelEvent<'_> {
 }
 pub struct KeyboardEvent<'a> {
     pub id: crate::Uuid,
-    pub namui_context: &'a NamuiContext,
+    // pub namui_context: &'a NamuiContext,
```
Comment (bigfoodK) [inline namui/src/namui/render/special/attach_event.rs]: preserved? or forgot?

**Review by bigfoodK — APPROVED** [review]

File: luda-adventure/src/app/game/game.rs [hunk 2]
```diff
@@ -69,6 +72,36 @@ impl Game {
...
+pub enum Event {
+    KeyDown {
+        code: Code,
+        pressing_codes: HashSet<Code>,
+    },
+    KeyUp {
+        code: Code,
+        pressing_codes: HashSet<Code>,
+    },
 }
```
[the remark this answers is not in the document]
Comment (bigfoodK) [reply luda-adventure/src/app/game/game.rs:107]: I think it's the right place.

File: luda-adventure/src/app/game/game.rs [hunk 3]
```diff
@@ -69,6 +72,36 @@ impl Game {
                 ]),
             ),
             self.menu.render(),
+            self.key_handler(),
         ])
     }
+
+    fn key_handler(&self) -> RenderingTree {
+        RenderingTree::Empty.attach_event(|builder| {
```
[the remark this answers is not in the document]
Comment (bigfoodK) [reply luda-adventure/src/app/game/game.rs:80]: I fixed some code. Thank you for mention.

File: namui/src/namui/render/special/attach_event.rs (hunk 1 above)
Comment (namse) [reply namui/src/namui/render/special/attach_event.rs]: Outdated

**Pull request merged**